1 table.sql

```diff
@@ -0,0 +1,28 @@
+drop TABLE if EXISTS dbo.Customer
+go
+CREATE table dbo.Customer(
+        CustomerID int not null IDENTITY primary key,
+        FirstName varchar(50) not null constraint ck_Customer_First_name_is_not_blank CHECK (FirstName <> ''), 
+        LastName varchar(50) not null constraint ck_Customer_Last_name_is_not_blank CHECK (LastName <> ''), 
+        PhoneNumber varchar(15) not null constraint ck_Customer_Phone_Number_is_not_blank CHECK (PhoneNumber <> ''), 
+        CustomerAddress VARCHAR(50) not null, constraint u_Customer_Address UNIQUE (CustomerAddress), constraint ck_Customer_Address_is_not_blank CHECK (CustomerAddress <> ''), 
```

Comment:
Remove the commas between column and constraints.

Reply:
done